view/frontend/web/js/view/payment/method-renderer/omise-offsite-installment-method.js

```diff
@@ -16,7 +16,10 @@ define(
         priceUtils
     ) {
         'use strict';
-        const INSTALLMENT_MIN_PURCHASE_AMOUNT = 2000;
+        const INSTALLMENT_MIN_PURCHASE_AMOUNT  = {
+            'thb': 2000,
+            'myr': 3000
```

Comment:
where did you get this value from? 3,000 MYR seems like a lot 🤔

Reply:
Oops. Changed it to 500. Thanks.